usr/lib/tik/modules/pre/10-welcome

```diff
@@ -3,34 +3,43 @@
 # SPDX-FileCopyrightText: Copyright 2024 Richard Brown
 # SPDX-FileCopyrightText: Copyright 2024 Raymond Yip
 
+displayWarning=false
+
 proceedInstall() {
 	d --info --ok-label="Install Now" --no-wrap --width=300 --height=300 --icon=distributor-logo-Aeon-symbolic --title="" --text="<big>Welcome to ${TIK_OS_NAME}</big>\n\nPlease press <b>Install Now</b> to continue"
 }
 
 warningMSG() {
```

Comment:
Again, I'd rename this function to reflect that it is about AC power. Also how about adding the verb, like `displayAcWarningMsg`.

Reply:
Fixed on next commit